Snowflake.Data/Core/SFBlockingChunkDownloaderV3.cs

```diff
@@ -95,8 +95,7 @@ private int GetPrefetchThreads(SFBaseResultSet resultSet)
 
         public Task<IResultChunk> GetNextChunkAsync()
         {
-            logger.InfoFmt("NextChunkToConsume: {0}, NextChunkToDownload: {1}",
-                nextChunkToConsumeIndex, nextChunkToDownloadIndex);
+            logger.Info($"NextChunkToConsume: {nextChunkToConsumeIndex}, NextChunkToDownload: {nextChunkToDownloadIndex}");

```

Comment:
Replaced InfoFmt() with equivalent Info() code since InfoFmt() and other Fmt() codes are removed to simplify the interface